SmartDeviceLink/public/SDLVideoStreamingCapability.h

```diff
@@ -30,7 +30,21 @@ NS_ASSUME_NONNULL_BEGIN
  @param scale The scaling factor the app should use to change the size of the projecting view
  @return A SDLVideoStreamingCapability object
  */
-- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale;
+- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale __deprecated_msg("Use initWithPreferredResolution:maxBitrate:supportedFormats:hapticDataSupported:diagonalScreenSize:pixelPerInch:scale:preferredFPS instead");
```

Comment:
This `init` is not implemented in the **.m** file (I think the new init was accidentally copied twice?). One of the test cases is failing due to this.

Reply:
Oops, I copied some `init` from generated SDLVideoStreamingCapability.m, but parameter set in generated method was different... Not sure why building the framework didn't fail on my XCode. My apologies.